en/firefox/new/desktop.ftl

```diff
@@ -0,0 +1,176 @@
+firefox-desktop-download-learn-about-the = Learn about the { -brand-name-firefox } desktop browser below
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-from-security-to = From security to news to gaming, there’s an <a { $attrs }>extension for everyone</a>. Add as many as you want until your browser is just right.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/about/
+firefox-desktop-download-firefox-was-created = { -brand-name-firefox } was created by <a {$attrs }>{ -brand-name-mozilla }</a> as a faster, more private alternative to browsers like Internet Explorer, and now { -brand-name-chrome }. Today, our mission-driven company and volunteer community continue to put your privacy above all else.
+
+# Variables:
+#   $attrs (attrs) - link to https://www.mozilla.org/en-US/firefox/privacy/
+firefox-desktop-download-as-the-internet = As the internet grows and changes, { -brand-name-firefox } continues to focus on your right to privacy  — we call it the <a { $attrs }>Personal Data Promise</a>: Take less. Keep it safe. No secrets. Your data, your web activity, your life online is protected with { -brand-name-firefox }.
+
+# Variables:
+#   $attrs (attrs) - link to https://accounts.firefox.com/signin
+firefox-desktop-download-firefox-is-available = { -brand-name-firefox } is available on all your devices; take your tabs, history and bookmarks with you. All you need is a <a { $attrs }>{ -brand-name-firefox } account</a>.
+
+# Variables:
+#   $attrs (attrs) - link to https://support.mozilla.org/en-US/kb/firefox-screenshots
+firefox-desktop-download-grab-a-high = Grab a high-resolution image of anything online with our <a { $attrs }>screenshot</a> tool built right in the browser
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-firefox-automatically = { -brand-name-firefox } automatically <a { $attrs }>blocks many third party trackers</a> from collecting and selling your web activity.
+
+# Variables:
+#   $attrs (attrs) - link to https://addons.mozilla.org/firefox/extensions/
+firefox-desktop-download-questions = Questions? <a { $attrs }>{ -brand-name-mozilla } support</a> has you covered.
+
+# The phrase “Now get even more from Firefox” is in specific reference to signing up for an account, which unlocks access to all our new products and services.
+firefox-desktop-download-youve-already-got-the-browser = You’ve already got the browser. Now get even more from { -brand-name-firefox }.
+firefox-desktop-download-watch-for-hackers-with = Watch for hackers with { -brand-name-firefox-monitor }, protect passwords with { -brand-name-firefox-lockwise }, and more.
+firefox-desktop-download-get-more-from-firefox = Get More From { -brand-name-firefox }
+firefox-desktop-download-just-download-the-browser = Just Download The Browser
+
+## URL: https://www-dev.allizom.org/firefox/new/thanks/
```

Comment:
I can't see the page. Any alternative link for review?

Reply:
This should be /firefox/download/thanks/